### ISSUE TITLE
Buttons "active" styling doesn't work

### ISSUE THREAD

**Issue by Pl4gue** [GeekyAnts/NativeBase#1587]
I tried the **Footer with only icons** example from the [docs](https://docs.nativebase.io/Components.html#footer-tabs-icon-headref), which should show the navigation icon as the only active button. 
But for me, **all buttons are shown active** - What's the issue?

## react-native, react and native-base version

-  **react-native-cli**: 2.0.1
-  **react-native**: 0.50.4
-  **react**: 16.0.0
-  **native-base**: ^2.3.8
 
## Expected behaviour
Only the navigation button should be active.
**Like this:**
![2018-02-14_13_47_58](https://user-images.githubusercontent.com/16862997/36205223-4eee36bc-118e-11e8-846f-e4bb52fbea9a.png)
## Actual behaviour
All buttons in the tabbar are shown active, as seen in the screenshot.

## Steps to reproduce (code snippet or screenshot)
```
      <Footer>
        <FooterTab>
          <Button>
            <Icon name="apps" />
          </Button>
          <Button>
            <Icon name="camera" />
          </Button>
          <Button active>
            <Icon active name="navigate" />
          </Button>
          <Button>
            <Icon name="person" />
          </Button>
        </FooterTab>
      </Footer>
```
## Screenshot of emulator/device
![Screenshot](https://lh5.googleusercontent.com/YyS3EwIlTt0X70WtjvkOxEoVngRoFxsY5upP6b4rIroqQLa9KkjyO5am1xNPhuIHBwiLHtFTLNisaPD9zRm7=w1676-h919-rw)
## Is the bug present in both ios and android or in any one of them?
**My device is a Huawei P8 running Android 6.1.**

**Comment by SupriyaKalghatgi** [comment]
@Pl4gue Can you upload screenshot of emulator / device?

**Comment by SupriyaKalghatgi** [comment]
@Pl4gue Tried your code snippet
```
<Footer>
  <FooterTab>
    <Button>
      <Icon name="apps" />
    </Button>
    <Button>
      <Icon name="camera" />
    </Button>
    <Button active>
      <Icon active name="navigate" />
    </Button>
    <Button>
      <Icon name="person" />
    </Button>
  </FooterTab>
</Footer>

```
And the output is here for [Android](https://github.com/GeekyAnts/NativeBase-KitchenSink/blob/master/screenshots/android/footer-icon.png) and [iOS](https://github.com/GeekyAnts/NativeBase-KitchenSink/blob/master/screenshots/ios/footer-icon.png)

Worked as expected

**Comment by Pl4gue** [comment]
I added the screenshot of my device @SupriyaKalghatgi

**Comment by SupriyaKalghatgi** [comment]
@Pl4gue The screenshot that you uploaded in issue description for **Screenshot of emulator/device** is broken

Yesterday I asked you to upload same

**Comment by Pl4gue** [comment]
Why is it still broken... -_- I will try again later

**Comment by SupriyaKalghatgi** [comment]
@Pl4gue Later? If that takes time, then you can close the issue
Since we presented you working  snippet with desired output

**Comment by Pl4gue** [comment]
![screenshot_2018-02-14-13-43-10](https://user-images.githubusercontent.com/16862997/36326007-ed7e4ac0-1359-11e8-9d62-7b08b45ca43b.png)
This is actually what it looks like on my device @SupriyaKalghatgi

**Comment by SupriyaKalghatgi** [comment]
Fixed with NB [2.3.10](https://github.com/GeekyAnts/NativeBase/releases/tag/v2.3.10)

**Comment by Alarees** [comment]
hi,
i have the same problem
i using `NB:2.13.8`,  `RN:0.61.5`
using emulator android 5.1
PS : i'm not using footer

```
            <View >
              <Button active >
                <Icon active name='md-call' >
              </Button>
            </View>
            <View >
              <Button >
                <Icon name='md-call' >
              </Button>
            </View>
            <View >
              <Button >
                <Icon name='md-call' >
              </Button>
            </View>
```
